### DISCUSSION TITLE
Lodestar Standup 2024-18 (May 7, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6728] [Standups]
### Meeting Info
[May 7, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/may-7-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - Pectra-devnet-0 Readiness
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by jeluard** [comment]
As I'm digging profiling/debugging and I'm about to write docs about those, I'm interested in understanding what tooling the team is using for both `coding`, `debugging` and `profiling`, and if they hit limitations/frustrations.

**Comment by philknows** [comment]
Closing for #6803
